Stack/Opc.Ua.Core/Schema/ApplicationConfiguration.cs

```diff
@@ -3260,7 +3261,7 @@ public UserIdentityToken UserIdentity
         /// The reverse connect information.
         /// </summary>
         [DataMember(Name = "ReverseConnect", Order = 8, IsRequired = false)]
-        public ReverseConnectEndpoint ReverseConnect 
+        public ReverseConnectEndpoint ReverseConnect
```

Comment:
Why not use auto-properties for all properties here? shorter:
public ReverseConnectEndpoint ReverseConnect { get; set; }

Reply:
thats not of concern in this PR